### PR TITLE
feat(nexus3): use termination log to record lifecycle errors

### PR DESCRIPTION
**Pull request by flah00**
I was chasing down some errors in my configuration. I thought this could help ease the task the next time I mess up my config.

### CODE REVIEW

File: charts/nexus3/files/configure.sh [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 set -eu
 
+function error() {
+  msg="ERROR $*"
+  >&2 echo "$msg"
+  echo "$msg" > "$TERMINATION_LOG"
+  exit 1
+}
+
+TERMINATION_LOG=${TERMINATION_LOG:-/dev/termination-log}
```
Comment (stevehipwell) [inline charts/nexus3/files/configure.sh:11]:
```suggestion
function error() {
  msg="ERROR: $*"
  >&2 echo "${msg}"
  echo "${msg}" > "${TERMINATION_LOG}"
  exit 1
}

TERMINATION_LOG="${TERMINATION_LOG:-/dev/termination-log}"
```

**Review by stevehipwell — CHANGES_REQUESTED** [review]
Thanks for the PR @flah00, I've added a suggestion to tweak your changes and could you add en entry to the CHANGELOG.

File: charts/nexus3/files/configure.sh (hunk 1 above)
Comment (flah00) [reply charts/nexus3/files/configure.sh]: new commit pushed.
thanks for the fantastically fast review!

**Review by stevehipwell — CHANGES_REQUESTED** [review]
Thanks for the changes @flah00. Sorry to make more work for you, but could you undo the changes to the chart YAML and leave the changes under `[UNRELEASED]` in the CHANGELOG?

**Comment by flah00** [comment]
just so i'm clear...
- don't bump nexus3 to `4.36.1`
- include the lifecycle script change in the changelog under the existing `4.36.0` release notes
is that right?

**Comment by stevehipwell** [comment]
@flah00 don't bump the chart version and add your changes under `[UNRELEASED]`.

**Review by stevehipwell — APPROVED** [review]
LGTM

**Pull request merged**